labcontrol/db/support_files/patches/python_patches/2.py

```diff
@@ -0,0 +1,90 @@
+from labcontrol.db.sql_connection import TRN
+import logging
+
+
+with TRN:
```

Comment:
If I understand correctly, these codes are to resolve some historical issue of naming inconsistency. Is that right?

Reply:
right. There was some confusion between the proper name, and the Greek letter kappa.